### DISCUSSION TITLE
Typhoon 5kg Coffee roaster with Artisan

### DISCUSSION THREAD

**Discussion by Danitocoffee** [artisan-roaster-scope/artisan#1292] [Q&A]
Hi folks. I'm starting with a Typhoon 5kg coffee roaster and wonder, if Artisan is supporting Typhoon fully or is it available with limitations? I found different statements and information so I hope to get some support within this community. Thanks in advance, Daniel

**Comment by MAKOMO** [comment]
Typhoon roasting machines are listed under [Supported Machines](https://artisan-scope.org/machines/index) and are supported by the current Artisan version.
